projects/1.20/assets/tetra/tetra/lang/zh_cn.json

```
@@ -0,0 +1,2465 @@
+  "tetra.tool.pry.craft_requirement": "加工需要一个%d级撬",
+  "tetra.tool.pry.material_requirement": "加工此材料的基础要求是一个%d级撬，此要求可能会因材料的用途不同而不同\n§7羊角头可用于撬",
+  "tetra.tool.available": "你有一个%d级的",
+  "tetra.stats.efficiency_suffix": "效率",
+  "tetra.stats.strength_suffix": "力量",
+  "tetra.stats.tier_suffix": "等级",
+  "tetra.stats.level_suffix": "等级",
+  "tetra.stats.reach": "触及范围",
+  "tetra.stats.reach.tooltip": "方块触及范围改变量：§e%s§r格。触及实体的距离不受影响",
+  "tetra.stats.attack_range": "攻击范围",
+  "tetra.stats.attack_range.tooltip": "实体触及范围改变量：§e%s§r格",
+  "tetra.stats.durability": "耐久度",
+  "tetra.stats.durability.tooltip": "此物品有§e%s§r点耐久度",
+  "tetra.stats.durability.tooltip_extended": "§7通常物品每次使用会消耗1或2点耐久度，当耐久度降到0时，物品就会破损，必须在加工台上修复才能再次使用",
+  "tetra.stats.speed": "速度",
+  "tetra.stats.speed.tooltip": "影响物品的挥动速度\n \n§7挥动间隔§e%s§7秒\n§7采集效率§e%s§7倍",
+  "tetra.stats.speed.tooltip_extended": "挥动“冷却”由屏幕中央的指示条显示。可以在冷却完毕前挥动，但如此攻击造成的伤害较少。横扫、急速打击等效果只有在攻击指示条完全充满时才会被触发\n \n§7速度是采集效率的倍数，它会加快或减缓方块被破坏的速度。倍数计算方式如下：§f§o倍数 = 速度 * 0.5 + 0.5§r§7",
+  "tetra.stats.draw_speed": "拉弓耗时",
+  "tetra.stats.draw_speed.tooltip": "拉满此物品需要§e%s§r秒",
+  "tetra.stats.draw_speed_normalized": "拉弓耗时",
+  "tetra.stats.draw_speed_normalized.tooltip": "拉弓耗时改变量：§e%s§r秒",
+  "tetra.stats.ability_speed": "冷却",
+  "tetra.stats.ability_speed.tooltip": "此物品能力的基础冷却时间为§e%s§r秒",
+  "tetra.stats.ability_speed_normalized": "冷却",
+  "tetra.stats.ability_speed_normalized.tooltip": "基础冷却时间改变量：§e%s§r秒",
+  "tetra.stats.speed_normalized": "速度",
+  "tetra.stats.speed_normalized.tooltip": "攻击速度改变量：§e%s",
+  "tetra.stats.attack_damage": "伤害",
+  "tetra.stats.attack_damage.tooltip": "此物品攻击会造成§e%s§r点伤害",
+  "tetra.stats.attack_damage.tooltip_extended": "物品能力造成的伤害通常基于物品本身的攻击伤害",
+  "tetra.stats.attack_damage_normalized": "伤害",
+  "tetra.stats.attack_damage_normalized.tooltip": "攻击伤害改变量：§e%s§r点",
+  "tetra.stats.draw_strength": "力量",
+  "tetra.stats.draw_strength.tooltip": "拉满时射出的弹射物会造成§e%s~%s§r点伤害，其初速度为§e%s§r格每刻",
+  "tetra.stats.draw_strength.tooltip_extended": "伤害计算基于原版的弹射物行为。伤害的下限取决于组件与改进，上限计算方法如下：§f§o上限 = 下限 * 1.5 + 1§r§7。由未拉满的弓射出的弹射物伤害基于下限计算，例如：从拉开50%%的弓射出的弹射物会造成50%%伤害下限的伤害",
+  "tetra.stats.power": "力量魔咒",
+  "tetra.stats.power.tooltip": "通过魔咒将弹射物的伤害增加§e%s§r，弹射物速度不受影响",
+  "tetra.stats.quick_charge": "快速装填魔咒",
+  "tetra.stats.quick_charge.tooltip": "通过魔咒将装填时间缩短§e%s§r秒",
+  "tetra.stats.ability_damage": "伤害",
+  "tetra.stats.ability_damage.tooltip": "此物品能力的基础伤害为§e%s§r点",
+  "tetra.stats.bleeding": "流血",
+  "tetra.stats.bleeding.tooltip": "攻击实体时有30%%的概率使其流血2秒，共造成§e%s§r点伤害，对亡灵生物无效",
+  "tetra.stats.backstab": "背刺",
+  "tetra.stats.backstab.tooltip": "从背后击中目标会产生暴击，并造成§e%s§r点额外伤害",
+  "tetra.stats.armor": "护甲",
+  "tetra.stats.armor.tooltip": "在持有时提供§e%s§r点护甲值",
+  "tetra.stats.toughness": "韧性",
+  "tetra.stats.toughness.tooltip": "在持有时提供§e%s§r点盔甲韧性",
+  "tetra.stats.shieldbreaker": "破盾",
+  "tetra.stats.shieldbreaker.tooltip": "攻击举起盾牌的实体时有§e25%%§r概率禁用其盾牌§e5§r秒（或其模块化盾牌基础冷却时间的§e75%%§r）",

```

| **审查级别**    |
|:-:|
| ❓疑问 / 需要作者确认 |

**原文: Attacking a blocking entity has a §e25%%§r chance to disable its shield for §e5§r seconds (or §e75%%§r of the base cooldown for modular shields)**

```suggestion
  "tetra.stats.shieldbreaker.tooltip": "攻击举起盾牌的实体时有§e25%%§r概率禁用其盾牌§e5§r秒（或其模块化盾牌基础冷却时间的§e75%%§r）",
```